### ISSUE TITLE
A grey(#cccccc) line between slide acitons

### ISSUE THREAD

**Issue by etanxing** [letsar/flutter_slidable#115]
![Simulator Screen Shot - iPhone 8 - 2020-01-03 at 20 36 23](https://user-images.githubusercontent.com/905634/71720392-7d80da00-2e71-11ea-8e62-ead506922c6f.png)

```
Widget createIconWidget(
        {Color color,
        Color foregroundColor,
        IconData icon,
        String caption}) =>
    Center(
        child: Container(
            color: Theme.of(context).scaffoldBackgroundColor,
            constraints: BoxConstraints.expand(),
            child: Container(
              margin: EdgeInsets.symmetric(vertical: 4.0),
              color: color,
              child: Column(
                mainAxisAlignment: MainAxisAlignment.center,
                children: <Widget>[
                  Icon(
                    icon,
                    size: 32.0,
                    color: foregroundColor,
                  ),
                  Text(
                    caption,
                    style: TextStyle(
                        fontSize: 12.0,
                        color: foregroundColor,
                        fontWeight: FontWeight.w600),
                  )
                ],
              ),
            )));

List<IconSlideAction> slideActions = [
  IconSlideAction(
      iconWidget: createIconWidget(
          color: orangeColor,
          foregroundColor: Colors.white,
          caption: 'STAR',
          icon: Icons.star),
      onTap: () {}),
  IconSlideAction(
      onTap: () {},
      iconWidget: createIconWidget(
          color: purpleColor,
          foregroundColor: Colors.white,
          caption: 'SHARE',
          icon: Icons.share))
];

child = Slidable(
    controller: widget._slidableController,
    actionPane: SlidableScrollActionPane(),
    actionExtentRatio: 0.25,
    secondaryActions: slideActions,
    child: child);
```

**Comment by etanxing** [comment]
I'm not sure which widget renders that line in `SlidableScrollActionPane`.

**Comment by Victor-Dev77** [comment]
It's the super widget. Even if we use `SlideAction` for more customization, you can't set the background color of the parent widget... So for example the `borderRadius` of `decoration` property is useless
![screen_demo](https://user-images.githubusercontent.com/42203616/76907447-86951b00-68a6-11ea-993e-8fba0f408bb6.PNG)

**Comment by theolm** [comment]
I have the same problem. Any workaround?

**Comment by barees63** [comment]
It would be very useful to be able to set the background color so we can have rounded corners, eg. in my current project we use slidable on cards which have rounded corners.

**Comment by letsar** [comment]
This has been fixed in 0.5.7
